src/spiceypy/spiceypy.py

```diff
@@ -14242,6 +14310,57 @@ def trcoff() -> None:
     libspice.trcoff_c()
 
 
+@spice_error_check
+def trgsep(
+    et: float,
+    targ1: str,
+    shape1: str,
+    frame1: str,
+    targ2: str,
+    shape2: str,
+    frame2: str,
+    obsrvr: str,
+    abcorr: str,
+) -> float:
+    """
+    Compute the angular separation in radians between two spherical
+    or point objects.
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/trgsep_c.html
+
+    :param et: Ephemeris seconds past J2000 TDB.
+    :param targ1: First target body name.
+    :param shape1: First target body shape.
+    :param frame1: Reference frame of first target.
+    :param targ2: Second target body name.
+    :param shape2: First target body shape.
+    :param frame2: Reference frame of second target.
+    :param obsrvr: Observing body name.
+    :param abcorr: Aberration corrections flag.
+    :return: angular separation in radians.
+    """
+    _et = ctypes.c_double(et)
+    _targ1 = stypes.string_to_char_p(targ1)
+    _shape1 = stypes.string_to_char_p(shape1)
+    _frame1 = stypes.string_to_char_p(frame1)
```

Comment:
`frame1` (`frame2`) are currently not used: the argument exists for future shape models. Maybe it'd be worth setting a default value, e.g. `'NULL'` as indicated in the CSPICE documentation?

Also, the user may input an empty string, which will cause CSPICE to complain as it checks for non-null strings with at least one character (+ the null terminator). Maybe it is worth checking if that's the case, and passing a single blank character string, i.e. if `frame1 = ''` then `_frame1 = ' '`. Same would apply to `frame2`.

Reply:
having two positional arguments with default values is hard to do, I'll leave them and just ignore them to always set `_frame` to `NULL` for now. I assume the empty string comment is just for this function so setting to NULL avoids that. If a user does supply an empty string intentionally or not the CSPICE error would be desireable